packages/flutter_tools/lib/src/commands/test.dart

```diff
@@ -47,7 +47,6 @@ class TestCommand extends FlutterCommand {
   @override
   bool get requiresProjectRoot => false;
 
-  @override
```

Comment:
This override was stale unless the field should actually be `commandValidator`?

@Hixie ?